content/documentation/glossary/index.adoc

```diff
@@ -0,0 +1,207 @@
+---
+title: "Glossary"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a logical group of <<Workload>> defined by the <<Label App>>.
+
+=== Application Name
+
+It's the name of the <<Application>> deployed in your environment. This name is provided by the <<App Label>> on the <<Workload>>
+
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
+link:https://istio.io/help/glossary/#deployment[Istio Glossary]
+
+== Destination
+
+The remote upstream service Envoy is talking to on behalf of a source workload. There can be one or more service versions for a given service and Envoy chooses the version based on routing rules.
+
+link:https://istio.io/help/glossary/#destination[Istio Glossary]
+
+=== Destination Rule
+
+DestinationRule defines policies that apply to traffic intended for a service after routing has occurred. These rules specify configuration for load balancing, connection pool size from the sidecar, and outlier detection settings to detect and evict unhealthy hosts from the load balancing pool
+link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#DestinationRule[Istio Destination Rule]
+
+
+== Endpoints
+
+A communication endpoint is a type of communication network node. It is an interface exposed by a communicating party or by a communication channel
+
+== Envoy
+
+The high-performance proxy that Istio uses to mediate inbound and outbound traffic for all services in the service mesh. Learn more about Envoy.
+link:https://istio.io/help/glossary/#envoy[Istio Glossary]
+
+== Envoy Health
+
+It's the status of the incoming and outcoming traffic.
+
+== Error Rate
+
+it's the percentage the errors in the traffic to a specific object in a <<Rate Interval>>
+
+== Gateway
+
+Gateway describes a load balancer operating at the edge of the mesh receiving incoming or outgoing HTTP/TCP connections. The specification describes a set of ports that should be exposed, the type of protocol to use, SNI configuration for the load balancer, etc.
+link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Gateway[Istio Gateway]
+
+== Inbound Metrics
+
+Incoming Metrics to a <<Workload>>, <<Service>> or <<Application>>
+
+== Istio Config
+
+It's a configuration object in a Istio environment
+
+=== Istio Name
+
+It's the name of the <<istio Config>>
+
+=== Istio Type
```

Comment:
shouldn't this be Istio object/configuration type? It is not Istio itself that has types